content/en/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md

```diff
@@ -378,6 +422,12 @@ and only enabled when the `DRAResourceClaimDeviceStatus`
 [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
 is enabled in the kube-apiserver.
 
+### Enabling Prioritized List
+
+[Prioritized List](#prioritized-list)) is an *alpha feature* and only enabled when the
+`DRAPrioritizedList` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+is enabled in the kube-apiserver and kube-scheduler.
```

Comment:
Please confirm we don't need to restart all `kubelets` for this feature to be effective.

Reply:
Better mention that `DynamicResourceAllocation` has to be enabled as well.